Suggested changes to Docs restructuring (#50)
Lumi-supercomputer/lumi-userguide: docs/eap/index.md

4. Request 2 tasks. A task in Slurm speaks is a process. If your application
   use MPI, it corresponds to the number of MPI ranks.

5. Request 8 threads per task. If your application is multithreaded (for
   example, using OpenMP) this is how you control the number of threads.

6. Request 2 GPUs for this job, one for each task. Most of the time the number
   of GPUs is the same as the number of tasks (MPI ranks).

7. If your application is multithreaded with OpenMP, set the value of
   `OMP_NUM_THREADS` to the value set with `--cpus-per-task`

8. If your code needs a GPU-aware MPI

> 9. Launch your application with `srun`. They are no `mpirun`/`mpiexec` on LUMI.

Reviewer: ```suggestion
9. Launch your application with `srun`. The `mpirun`/`mpiexec` do not work on LUMI.
```

Author: I went with `There are no"
